### DISCUSSION TITLE
Error: emineJ

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#312] [Q&A] [answered]
I followed the steps in the book, but made the following mistake:

![](https://cdn.discordapp.com/attachments/729301535082676255/1184561202798530711/Honeycam_2023-12-14_02-20-00.gif)

Then the program will wait indefinitely, [and my interface is very different from the the book.](https://www.biostarhandbook.com/using-the-erminej-tool.html#:~:text=When%20we%20start%20up%20ErmineJ%2C%20the%20following%20page%20will%20be%20presented%3A)

**Comment by ialbert** [comment, marked as answer]
I think I will rework that chapter, lots have changed with the tools in the years since it was written

it is completely out of our control wether the tool still works or not,

**Reply by TullyMonster** [reply]
So far, are there GUI tools that might replace `emineJ`?
